modules/mcagar/src/main/ts/ephox/mcagar/alien/EditorTypes.ts

```diff
@@ -51,6 +56,7 @@ export interface Editor {
 
   getContent: (args?: Partial<GetContentArgs>) => string;
   setContent: (content: string, args?: Partial<SetContentArgs>) => void;
+  content: Content;
```

Comment:
This will need to be optional, as these types need to work for all TinyMCE versions, not just the latest version.

On that note, does this actually need to be added to the common editor type in McAgar? I don't believe it does since McAgar doesn't make use of it.